delivery_package_number/wizard/stock_backorder_confirmation_views.xml

```diff
@@ -7,19 +7,19 @@
         <field name="arch" type="xml">
             <xpath expr="//footer" position="before">
                 <field name="ask_number_of_packages" invisible="1" />
-                <group attrs="{'invisible': [('ask_number_of_packages','=',False)]}">
+                <group invisible="ask_number_of_packages == False">
```

Comment:
I like to rely on coercion of truthy / falsey values. I.e.: `None` is falsey but isn't equal to `False`. It's also more semantic
```suggestion
                <group invisible="not ask_number_of_packages">